### ISSUE TITLE
Request to add the pauseAll() or GetWaitingDownload() function to libaria2

### ISSUE THREAD

**Issue by xiaoke0O** [aria2/aria2#1886]
I know libaria2 has the getActiveDownload() function and the GlobalStat::numWaiting method, and I've looked at the source code for both, so I'm curious if it's possible to get numWaiting. Why not implement the getWaitingDownload() function, just like getActiveDownload().

**Comment by zjyl1994** [comment]
+1, I also need this functionality, currently libaria2 provides less functionality than RPC mode.
I don't know what is the process for submitting a libaria2 change? Need to fork first?

**Comment by xiaoke0O** [comment]
> +1, I also need this functionality, currently libaria2 provides less functionality than RPC mode. I don't know what is the process for submitting a libaria2 change? Need to fork first?

Yes, fork first, then modify the code in your own repository and commit, and finally submit the pull request on the browser side.
